Source/Core/Core/CMakeLists.txt

```diff
@@ -684,6 +684,7 @@ if(WIN32)
   target_sources(core PRIVATE
     HW/EXI/BBA/TAP_Win32.cpp
     HW/EXI/BBA/TAP_Win32.h
+    HW/EXI/BBA/TAPServer.cpp
     HW/EXI/BBA/XLINK_KAI_BBA.cpp
     HW/EXI/BBA/BuiltIn.cpp
     HW/EXI/BBA/BuiltIn.h
```

Comment:
These are now listed on all platforms. XLINK_KAI_BBA and BuiltIn were already duplicated like this, but it'd probably be better to move it earlier on, between these lines:

https://github.com/dolphin-emu/dolphin/blob/8fe777c1acb6a828264d377dc5ba504736448944/Source/Core/Core/CMakeLists.txt#L189-L190

Reply:
Thanks; will change this.